ledger/participant-integration-api/src/main/scala/platform/apiserver/configuration/RateLimitingConfig.scala

```diff
@@ -18,10 +18,11 @@ package com.daml.platform.apiserver.configuration
   *   this value the system will be rate limited until additional space is freed up.
   */
 final case class RateLimitingConfig(
-    maxApiServicesQueueSize: Int,
-    maxApiServicesIndexDbQueueSize: Int,
-    maxUsedHeapSpacePercentage: Int,
-    minFreeHeapSpaceBytes: Long,
+    enabled: Boolean = true,
```

Comment:
The global default for rate limiting should be off (false) as it is not yet feature complete.

It should be on (true) for Sandbox-on-X only.

But I see that you are providing a way, when rate limiting is on by default, to turn it off.  I may change the RateLimitConfig into a trait which which can be Off or On (with params).